src/gbd_mapping/cause_template.py

```diff
@@ -14,10 +14,11 @@
 
 class Cause(ModelableEntity):
     """Container for cause GBD ids and metadata"""
-    __slots__ = ('name', 'gbd_id', 'dismod_id', 'restrictions', 'sequelae', 'etiologies', )
+    __slots__ = ('name', 'type', 'gbd_id', 'dismod_id', 'restrictions', 'sequelae', 'etiologies', )
 
     def __init__(self,
                  name: str,
+                 type: str,
```

Comment:
Use a different argument name.  `type` is a python builtin.  The class can still have `type` attribute, but we don't want to muck up the namespace.